### PR TITLE
Added stub for user module

### PR DESCRIPTION
**Pull request by ashwch**
Related to: https://github.com/python/typeshed/issues/1147

### CODE REVIEW

**Comment by JelleZijlstra** [comment]
This seems like a bit of a special case, in that it's intended to include arbitrary user-specified code. I think we should add a module-level `__getattr__` as agreed on in #5.

File: stdlib/2/user.pyi [hunk 1]
```diff
@@ -0,0 +1,10 @@
+# Stubs for user (Python 2)
+
+# Docs: https://docs.python.org/2/library/user.html
+# Source: https://hg.python.org/cpython/file/2.7/Lib/user.py
+from typing import BinaryIO, Text, Union
+
+
+home: str
+pythonrc: Union[bytes, Text]
```
Comment (JelleZijlstra) [inline stdlib/2/user.pyi]: Isn't it always bytes?

File: stdlib/2/user.pyi [hunk 2]
```diff
@@ -0,0 +1,10 @@
...
+f: BinaryIO
```
Comment (JelleZijlstra) [inline stdlib/2/user.pyi]: This may not exist (it will only exist if ~/.pythonrc.py exists).

File: stdlib/2/user.pyi (hunk 1 above)
Comment (ashwch) [inline stdlib/2/user.pyi]: In this case yes, but it can have multiple [return both](https://github.com/python/typeshed/blob/master/stdlib/2/os/path.pyi#L62) in general.

File: stdlib/2/user.pyi (hunk 2 above)
Comment (ashwch) [inline stdlib/2/user.pyi]: @JelleZijlstra How do we deal with variables that may or may not exist?

Secondly is it okay if I include few more stubs in the same PR?

File: stdlib/2/user.pyi (hunk 1 above)
Comment (JelleZijlstra) [inline stdlib/2/user.pyi]: Well, in this case all the arguments are bytes, so the return will also be bytes.

File: stdlib/2/user.pyi (hunk 2 above)
Comment (JelleZijlstra) [inline stdlib/2/user.pyi]: I think the module-level `__getattr__` should suffice and we can just omit `f`.

Comment (JelleZijlstra) [inline stdlib/2/user.pyi]: Also, please submit stubs for other modules as separate PRs.

**Comment by JelleZijlstra** [comment]
To fix the test failure, you can add `# type: ignore` for now. I just submitted a PR to mypy (python/mypy#3647) to support module-level `__getattr__` properly.
